recipe/build.sh

```diff
@@ -1,4 +1,4 @@
 # Need to add the openjpeg2 cflags
 export CFLAGS="${CFLAGS} $(pkg-config --cflags libopenjp2)"
 
-$PYTHON -m pip install . -vvv --no-deps
+$PYTHON -m pip install . --no-deps --no-build-isolation
```

Comment:
```suggestion
$PYTHON -m pip install . --no-deps --no-build-isolation -v
```
`-v` will let us see the build logs.